bot/exts/moderation/voice_gate.py

```diff
@@ -258,19 +210,24 @@ async def on_voice_state_update(self, member: Member, before: VoiceState, after:
 
         # To avoid race conditions, checking if the user should receive a notification
         # and sending it if appropriate is delegated to an atomic helper
-        notification_sent, message_channel = await self._ping_newcomer(member)
-
-        # Schedule the channel ping notification to be deleted after the configured delay, which is
-        # again delegated to an atomic helper
-        if notification_sent and isinstance(message_channel, discord.TextChannel):
-            await asyncio.sleep(GateConf.voice_ping_delete_delay)
-            await self._delete_ping(member.id)
+        await self._ping_newcomer(member)
 
     async def cog_command_error(self, ctx: Context, error: Exception) -> None:
         """Check for & ignore any InWhitelistCheckFailure."""
         if isinstance(error, InWhitelistCheckFailure):
             error.handled = True
 
+    @command(name="prepare_voice")
+    @has_any_role(*MODERATION_ROLES)
+    async def prepare_voice_button(self, ctx: Context, channel: TextChannel | None, *, text: str) -> None:
```

Comment:
Also, the `text` part doesn't change AFAIK since it'll always have the same rules. So how about remove the burden of writing it out in Discord and have it saved as a constant ?

Reply:
The wording may change and other things may be added in the future.  Just seemed convenient at the time.

Reply:
I've always thought it was just like the incidents channel, where the message is sent by a regular user rather than a bot user.

That way we get rid of these constrains.

Reply:
I think in incidents it's fine since we just have reactions, but we can't add a button to a user-sent message, so this unfortunately has to be sent by the bot

Reply:
I think what we could do is if the user invoking this command omits the channel, then the default channel would be `#voice-verify`. If they specify a channel, the message would go there. 

Same deal with the text, maybe we could set the fallback text value to what we already have, but it can be overridden if necessary.

@MrHemlock I think we'll also either need to delete the original message that invoked the command if it's sent in `#voice-verify` so the initial command message doesn't linger

Reply:
I was thinking we would be doing it in a channel like mod-spam or admin-spam.  And we'd have the ability to just delete the message itself.  Not only that, not deleting it right away makes it easy to re-copy and fix if I (inevitably) type something incorrectly while putting up the rules and message.  I feel like this command really doesn't need to have a lot of functionality tied up with it.  It's only going to be used once in a blue moon, likely just by me.

And yes, because it's adding a view, it has to come from a bot.